src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,29 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class BulldozerProducer<T extends Bulldozer> implements MachineProducer<T> {
+    private final String[] colors = new String[]{"red", "blue", "black",
+            "orange", "grey", "green", "yellow"};
+    private final String[] names = new String[]{"Volvo", "Freightliner", "Peterbilt",
+            "JCB", "KOMATSU", "HITACHI", "CAT"};
+    private final String[] chassis = new String[]{"crawler", "wheeled"};
+    private final Random random = new Random();
+
+    @Override
+    public List<? extends Machine> get() {

```

Comment:
```suggestion
    public List<Bulldozer> get() {
```
так як це бульдозер продюсер він має генерити ліст бульдозерів